### ISSUE TITLE
print/highlight human readable message from log line

### ISSUE THREAD

**Issue by mac2000** [stern/stern#277]
created small tool for myself

![screenshot](https://github.com/mac2000/lens/blob/main/screenshot.png?raw=true)

http://github.com/mac2000/lens

nothing special, it was more about having fun

then was pointer about stern on [reddit](https://www.reddit.com/r/kubernetes/comments/160r9po/comment/jxpmbt0/?utm_source=share&utm_medium=web2x&context=3)

pretty cool project 💪

wondering if thats will be possible to do something similar - aka try to extract human readable message from log line and print it alone

otherwise it is so hard to scan logs by eyes - it was exactly the reason for tool

i see there are templates, but I bed no one will have collection of templates for all use cases / patterns

**Comment by superbrothers** [comment]
I think we should consider another tool for this kind of thing to keep stern simple.

**Comment by mac2000** [comment]
Yes, it is just an idea and totally depends on what we want to keep simple - codebase or usage

In ideal world I am imagining this as just addition of some library that already doing this and one more flag, which will fit both criteria

but if that’s not a case then probably indeed it may be dedicated tool so it may be reused with other apps

also I was wondering if that feature will collect some feedback and interest in general but seems like it did not so probably it is safe to say it is not needed at this time

so I am closing this to not bother you guys

and once again thank you for cool tool 💪
